src/Configuration/Cache/IlluminateCacheAdapter.php

```diff
@@ -54,7 +54,11 @@ protected function doSave($id, $data, $lifeTime = false)
             return $this->cache->forever($id, $data);
         }
 
-        return $this->cache->put($id, $data, $lifeTime);
+        // Laravel cache system accept expire times in minutes 
+        // while Doctrine accept it in seconds, so we need to convert $lifeTime to minutes
+        // before passing it to Laravel cache repository
+        $lifeTime_minutes = $lifeTime/60;
```

Comment:
We are still using php-cs-fixer v1. 

You can fix it manually be checking: https://styleci.io/analyses/qrJVRD